spec/unit/recipes/_configure_indexes_spec.rb

```diff
@@ -134,6 +134,91 @@
         }
       }
 
+      expect(subject).to create_splunk_template('system/indexes.conf').with(expected_attributes)
+    end
+  end
+  # When _noGenerateTstatsHomePath => true in specific indexes, then writes the indexes.conf file without tstats for only for that indexes
+  context 'when _noGenerateTstatsHomePath is set to true' do
+    let(:index_config) do
+      {
+        'config' => {
+          'volume:test' => {
+            'path' => '/test/path'
+          },
+          'index_a' => { '_directory_name' => 'foo', '_noGenerateTstatsHomePath' => true },
```

Comment:
if `_volume` is not set, then `tstatsHomePath` will not get set[1]. So you need to add a `_volume` attribute to really test `_noGenerateTstatsHomePath`. Same applies to the other test.
[1] https://github.com/cerner/cerner_splunk/pull/182/files#diff-79be619f4d42bccfc1205531edfcace2R66

Reply:
updated _volume. Thanks